keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -833,26 +798,28 @@ Possible consequences are:
 
 ###### What specific metrics should inform a rollback?
 
-TODO
+- `apiserver_authentication_config_controller_automatic_reload_failures_total` - This metric will be incremented when the API server fails to reload the configuration file.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-TODO
+This will be covered by integration tests.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-TODO
+No.
 
 ### Monitoring Requirements
 
-<!--
-This section must be completed when targeting beta to a release.
+New metrics:
 
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
--->
-
-TBA
+- `apiserver_authentication_config_controller_automatic_reload_failures_total` - This metric will be incremented when the API server fails to reload the configuration file.
+- `apiserver_authentication_config_controller_automatic_reload_last_timestamp_seconds` - This metric will be updated every time the API server reloads the configuration file.
```

Comment:
The hash of the config file itself could be useful?

Reply:
The hash can be useful. `apiserver_authentication_config_controller_automatic_reload_last_timestamp_seconds` is set every time we attempt the reload of the configuration with the `status` label. We only might want to add the hash of config in the success path, so we could have a separate metric similar to `apiserver_authentication_last_jwks_fetch_success_timestamp_seconds`. WDYT?

Reply:
SGTM.

Reply:
added a new metric `apiserver_authentication_config_controller_automatic_reload_last_success_timestamp_seconds`